src/main/scala/uk/gov/nationalarchives/tdr/api/db/repository/ConsignmentRepository.scala

```diff
@@ -25,13 +23,22 @@ class ConsignmentRepository(db: Database) {
     db.run(query.result)
   }
 
+  def getSeriesAndBodyOfConsignment(consignmentId: UUID)(implicit executionContext: ExecutionContext): Future[Seq[ConsignmentResult]] = {
+    val query = Consignment.join(Series)
+      .on(_.seriesid === _.seriesid).join(Body)
+      .on(_._2.bodyid === _.bodyid)
+      .filter(_._1._1.consignmentid === consignmentId)
+      .map(rows => (rows._1._1, rows._1._2, rows._2))
+    db.run(query.result).map(rows => rows.map(result => ConsignmentResult(result._1, result._2, result._3)))
```

Comment:
Does the consignment need to be returned at all? I don't think it's being used in the service.

Reply:
No, it doesn't; I'll remove it. I thought since it's being returned by the database, it might as well be returned from the method but I guess if it comes in handy in the future, someone can add it back, they would only need to change the services that called it and that would probably be a small thing